src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/ConfigureEndpointRabbitMQTransport.cs

```diff
@@ -77,22 +74,23 @@ void PurgeQueues()
             throw new Exception("The connection string doesn't contain a value for 'host'.");
         }
 
-        var queues = queueBindings.ReceivingAddresses.Concat(queueBindings.SendingAddresses);
-
-        using (var connection = connectionFactory.CreateConnection("Test Queue Purger"))
-        using (var channel = connection.CreateModel())
-        {
-            foreach (var queue in queues)
-            {
-                try
-                {
-                    channel.QueuePurge(queue);
-                }
-                catch (Exception ex)
-                {
-                    Console.WriteLine("Unable to clear queue {0}: {1}", queue, ex);
-                }
-            }
-        }
+        //TODO: Cleanup
+        //var queues = queueBindings.ReceivingAddresses.Concat(queueBindings.SendingAddresses);
```

Comment:
Maybe this can be handled as [in SQL](https://github.com/Particular/NServiceBus.SqlServer/pull/743/files#diff-f3f090d3380d288229eb729f362e4a68497ef1703b3feeef8968e686af0bbacfR61) via internal for-tests only properties?

Reply:
👍 Added in the latest commit.